docs/protocol-design/introduction.md

```diff
@@ -49,6 +49,37 @@ Although there is cross-over between the two main areas of the living whitepaper
 | [Contributing](../node-implementation/contributing.md) | How to contribute to the Nano protocol directly | 
 | [Original Whitepaper](../whitepaper/english.md) | Online version of original whitepaper last revised in November 2017 | 
 
+## Background
+
+In 2008, an anonymous individual under the pseudonym Satoshi Nakamoto published a whitepaper outlining the world’s first decentralized cryptocurrency, Bitcoin [^3]. A key innovation brought about by Bitcoin was the blockchain, a public, immutable and decentralized data-structure which is used as a ledger for the currency’s transactions. Unfortunately, as Bitcoin matured, several issues in the protocol made Bitcoin prohibitive for many applications: 
+
+1. Poor scalability: Each block in the blockchain can store a limited amount of data, which means the system can only process so many transactions per second, making
+spots in a block a commodity. Median transaction fees flucutate between a few cents and as high as $34 (currently ~$2.98 as of August 26, 2020) [^4].
```

Comment:
Escape characters (`\`) needed before the dollar signs here to avoid formatting as math equations.

Reply:
Fixed!